cypher/cypher-docs/src/docs/dev/ql/administration/security/user-management-syntax-alter-user.asciidoc

```diff
@@ -2,11 +2,15 @@
 [source, cypher]
 -----
 ALTER USER name SET {
-      [PLAINTEXT | ENCRYPTED] PASSWORD password
+[PLAINTEXT \| ENCRYPTED] PASSWORD password
```

Comment:
I don't think we should have `\|` here, it is not in a table so it will print out both characters (when in tables the `|` has special meaning and is escaped with `\` to be printed instead)

This goes for all lines